firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionHeaderView.swift

```diff
@@ -114,9 +114,15 @@ final class TrackingProtectionHeaderView: UIView, ThemeApplicable {
     }
 
     func adjustLayout() {
-        let faviconSize = UX.faviconImageSize
-        faviconHeightConstraint?.constant = min(UIFontMetrics.default.scaledValue(for: faviconSize), 2 * faviconSize)
-        faviconWidthConstraint?.constant = min(UIFontMetrics.default.scaledValue(for: faviconSize), 2 * faviconSize)
+        let faviconDynamicSize = min(UIFontMetrics.default.scaledValue(for: UX.faviconImageSize),
+                                     2 * UX.faviconImageSize)
+        faviconHeightConstraint?.constant = faviconDynamicSize
+        faviconWidthConstraint?.constant = faviconDynamicSize
+
+        let closeButtonDynamicSize = min(UIFontMetrics.default.scaledValue(for: UX.closeButtonSize),
```

Comment:
Yes that was already in place. But i think it was done for smal font classes to avoid to have very small icons. Anyway that is done only here so i'm going to remove it. Everywhere else works fine withouth